libnetdata/inlined.h

```diff
@@ -246,6 +246,20 @@ static inline char *strncpyz(char *dst, const char *src, size_t n) {
     return p;
 }
 
+static inline void escape_json_string(char *dst, char *src, size_t len) {
```

Comment:
The result of this is being written into a resizing buffer, why are we not using it instead of a fixed-length temporary array?

Reply:
For label creation, you tried to avoid memory allocations sacrificing readability. Why do we need to use resizing buffer here, when these operations will take place much more frequently than label creation. Are we afraid of losing some characters in insanely long label values in API responses?

Reply:
At this point in the code we are building a response in a resizing buffer already so there is no sacrifice in performance.

Double the size of the buffer to handle the worst-case.

Reply:
Done